src/views/assets/Platform/PlatformDetail/Detail.vue

```diff
@@ -101,16 +101,16 @@ export default {
       const { object } = this
       const quickActions = [
         {
-          title: this.$t('EnableDomain'),
+          title: this.$t('EnableGateway'),
           type: 'switch',
           attrs: {
             label: this.$t('Update'),
-            model: object['domain_enabled'],
-            disabled: !this.canEdit || this.constraints['domain_enabled'] === false
+            model: object['gateway_enabled'],
+            disabled: !this.canEdit || this.constraints['gateway_enabled'] === false
           },
           callbacks: Object.freeze({
             change: (val) => {
-              const data = { domain_enabled: val }
+              const data = { gateway_enabled: val }
               this.$axios.patch(
                 `/api/v1/assets/platforms/${object.id}/`, data).then(res => {
                 this.$message.success(this.$tc('UpdateSuccessMsg'))
```

Comment:
There are no significant differences between the two versions of the code provided. Both contain correct usage of object properties and methods including accessing an endpoint, setting up data for components such as quick actions, enabling or disabling certain functionalities based on constraints conditions. No bugs, security vulnerabilities or performance improvements have been observed at this time.

As it is not clear which version(s) you are referring to, please specify either the original version before optimization changes, or both sides for a comprehensive view.